### ISSUE TITLE
Error Failed parsing: output/

### ISSUE THREAD

**Issue by dgendill** [coot/zephyr#7]
I thought I would try out Zephyr and see if I could improve my build sizes, but I seem to be running into a problem when I run `zephyr Main.main`. Here's what I did.

First I checked out the zephyr repo and ran `stack install`, which added zephyr to my path.  I have Stack Version 1.5.1 x86_64 hpack-0.17.1, PureScript 0.11.6, and Pulp 11.0.0. Then I went to my project's root and ran `pulp build -- --dump-corefn` which compiled my project. Then I ran `zephyr Main.main` and I got this error...

```
Error
Failed parsing:
  output/DOM.File.FileReader/corefn.json
        output/DOM.Event.WheelEvent/corefn.json
        output/Data.HeytingAlgebra/corefn.json
        output/DOM.HTML.HTMLMediaElement.NetworkState/corefn.json
        output/Control.Monad.Writer.Class/corefn.json
        output/Data.Bifunctor.Clown/corefn.json
        output/DOM.HTML.HTMLOptGroupElement/corefn.json
        output/Data.List.Lazy.NonEmpty/corefn.json
        output/Data.Distributive/corefn.json
        output/Data.Interval/corefn.json
        output/Control.Monad.Gen/corefn.json
        output/Data.Int.Bits/corefn.json
        output/Data.Monoid.Dual/corefn.json
        output/Data.Time.Component.Gen/corefn.json
        output/Data.Identity/corefn.json
        output/Data.Monoid/corefn.json
        output/DOM.HTML.HTMLTableElement/corefn.json
        output/Data.Char/corefn.json
        output/DOM.XHR.Types/corefn.json
        output/DOM.Node.NonElementParentNode/corefn.json
        output/Data.Traversable/corefn.json
        output/Control.Comonad.Store/corefn.json
        ....
```
I don't think there's anything unusual about the project I'm compiling. This is the [bower.json file](https://github.com/dgendill/purescript-keycombos/blob/master/bower.json) with my project dependencies.

**Comment by coot** [comment]
You need to compile with correct version of `purs`.  Grab the one that is compiled together with `zypher`.  It should be under `.stack-work/install/x86_64-linux/lts-8.5/8.0.2/bin` (or similar if you're on a different architecture).   It relies on yet unmerged PR, but I think it will be published together with `purs-0.12.0`.

**Comment by dgendill** [comment]
Thanks, using the version in the /bin folder worked!
